doc(intro/install): mention necessary SELinux changes

RobertCochran opened the pull request:
I noticed that a topic that poster Nikolai Shcherbin seems to address frequently
are questions regarding installing Elgg on CentOS. A lot of people seem to be
getting stuck at various points in the installation procedure, and the common
link appears to be SELinux-related permissions issues. It therefore seems
prescient to document what needs to be done in the official documentation to cut
down on the help requests.

-----

Please let me know if there are any silly errors this patch; I followed `CONTRIBUTING` to the best of my ability, but I'm ultimately unfamiliar with this toolchain.

jeabakker commented on docs/intro/install.rst:
```diff
@@ -38,6 +38,14 @@ By "official support", we mean that:
+.. note::
+
+   If using RHEL, CentOS, or any other distribution with SELinux enabled, you
+   will need to make sure that the appripriate contexts and permissions are configured:
```
there is a typo here `appripriate ` needs to be `appropriate`

RobertCochran replied:
Alas, spelling is hard. I fixed that, the commit message, and the extra file. For what it's worth, the documentation at http://learn.elgg.org/en/stable/contribute/docs.html#internationalizing-documentation implies (at least to me) that one *should* add the `.pot` file as part of the commit. If that's not right, maybe that area of documentation needs to be revised or clarified as to which situations do and do not need the translation files.

jeabakker reviewed (changes requested):
- the `.pot` file isn't necessary in the PR. the release process will build that file
- in the commit message you can't have a `/` so please change it to `doc(install):....`

RobertCochran committed:
doc(install): fix typo in Troubleshooting section

RobertCochran committed:
doc(install): mention necessary SELinux changes

I noticed that a topic that poster Nikolai Shcherbin seems to address frequently
are questions regarding installing Elgg on CentOS. A lot of people seem to be
getting stuck at various points in the installation procedure, and the common
link appears to be SELinux-related permissions issues. It therefore seems
prescient to document what needs to be done in the official documentation to cut
down on the help requests.

RobertCochran commented:
As part of fixing the typo originally pointed out, I ran a spell check over the file and found a second typo, so there's a second commit for that now.

jdalsem commented:
Thanks for the contribution! I've squashed your commits together (as they belong together for the same change).
